test.sh

```diff
@@ -119,6 +119,12 @@ RUN="$ENGINE exec -ti $CONTAINER_NAME"
     $RUN $PIP install configparser==4.0.2
   fi
 
+  # pyrsistent >= 0.17 no longer supports python 2
+  # pyrsistent is a dependency of jsonschema
+  if [[ $PYTHON_VERSION == 2 ]]; then
+    $RUN $PIP install 'pyrsistent<0.17'
```

Comment:
I'd install version `pyrsistent=0.16.*`, this may keep older version of system (by accident) and jsonchema will upgrade it again because it needs higher version than 14 and break it on py2
https://github.com/Julian/jsonschema/blob/master/setup.cfg#L31

Reply:
ack

Reply:
fixed